### PR TITLE
[Ripple] Fix active ripple color not having a persistent storage issue.

### PR DESCRIPTION
**Pull request by fluxminds**
rippleColor was overwriting the color value of activeRippleLayer because there was no way to store the real active ripple color.

There was a view logic dependency that activeRippleColor's value was initialized from rippleColor if there is no value set. The added getter persists that for consistence.

### CODE REVIEW

**Review by yarneo — APPROVED** [review]

File: components/Ripple/src/MDCRippleView.m [hunk 1]
```diff
@@ -157,13 +171,13 @@ - (void)beginRippleTouchDownAtPoint:(CGPoint)point
 #if defined(__IPHONE_13_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0)
   if (@available(iOS 13.0, *)) {
     [self.traitCollection performAsCurrentTraitCollection:^{
-      rippleLayer.fillColor = self.rippleColor.CGColor;
+      rippleLayer.fillColor = self.activeRippleColor.CGColor;
```
Comment (yarneo) [inline components/Ripple/src/MDCRippleView.m:174]: why was this updated? it should still use the rippleColor property here and below

File: components/Ripple/src/MDCStatefulRippleView.m [hunk 2]
```diff
@@ -144,15 +144,13 @@ - (void)setSelected:(BOOL)selected {
   _selected = selected;
   // Go into the selected state visually.
   if (selected) {
+    [self updateActiveRippleColor];
```
Comment (yarneo) [inline components/Ripple/src/MDCStatefulRippleView.m]: i wouldn't make any changes here as this code is pretty fragile, why were these changes needed?

**Review by yarneo — DISMISSED** [review]

File: components/Ripple/src/MDCStatefulRippleView.m (hunk 2 above)
Comment (fluxminds) [reply components/Ripple/src/MDCStatefulRippleView.m]: I think its gonna be fine if we remove this change, since I added the getter change. This change here was based the original logic. It was using ripple color as the value to set activeLayer before.

File: components/Ripple/src/MDCRippleView.m (hunk 1 above)
Comment (fluxminds) [reply components/Ripple/src/MDCRippleView.m:174]: This is the activeRippleLayer, I assume it should use the activeRippleColor, isn't it?

File: components/Ripple/src/MDCStatefulRippleView.m (hunk 2 above)
Comment (fluxminds) [reply components/Ripple/src/MDCStatefulRippleView.m]: I can move the change there to a separate PR. Though it would affect the test, I think it worth to clean up the fragile dependent logic here.

Comment (fluxminds) [reply components/Ripple/src/MDCStatefulRippleView.m]: Removed in 95206a3b5329c4009ec198ef11921a73d8358534.

**Review by yarneo — COMMENTED** [review]
Please see
https://github.com/material-components/material-components-ios/pull/7949
